pkg/repository/prisma/dbsqlc/workflow_runs.sql

```diff
@@ -1551,3 +1551,23 @@ WHERE
 DELETE FROM "WorkflowTriggerScheduledRef"
 WHERE
     "id" = @scheduleId::uuid;
+
+-- name: GetUpstreamErrorsForOnFailureStep :many
+WITH workflow_run AS (
+    SELECT wr."id"
+    FROM "WorkflowRun" wr
+    JOIN "JobRun" jr ON wr."id" = jr."workflowRunId"
+    JOIN "StepRun" sr ON jr."id" = sr."jobRunId"
+    WHERE sr."id" = @onFailureStepRunId::uuid
+)
+SELECT
+    sr."id" AS "stepRunId",
+    s."readableId" AS "stepReadableId",
+    sr."error" AS "stepRunError"
+FROM "WorkflowRun" wr
```

Comment:
Very minor nit -- making this `FROM workflow_run wr` would prevent you from having to do `WHERE wr."id" = (SELECT "id" FROM workflow_run)`